tsrc/executor.py

```diff
@@ -0,0 +1,71 @@
+""" Helpers to run thing on multiple repos and collect errors """
+
+import abc
+
+from tsrc import ui
+import tsrc
+
+
+class ExecutorFailed(tsrc.Error):
+    pass
+
+
+class Actor(metaclass=abc.ABCMeta):
```

Comment:
Or a Task.

I am asking cause I dont see signal/slot nor thrand associated to the actor class.